template/postcss.config.js

```diff
@@ -0,0 +1,26 @@
+const postcssConfig = {
+  plugins: [
+    require('postcss-import'),
+    require('tailwindcss'),
+    require('postcss-nested'),
+    require('postcss-custom-properties'),
+    require('autoprefixer'),
+  ],
+};
+
+if (process.env.NODE_ENV === 'production') {
+  postcssConfig.plugins = [
+    ...postcssConfig.plugins,
+    require('@fullhuman/postcss-purgecss')({
```

Comment:
The latest version of Tailwind CSS comes with Purge CSS, we can remove from the PostCSS configuration.